peachjam_ml/templates/peachjam/_similar_documents.html

```diff
@@ -1,5 +1,5 @@
 {% load i18n %}
-<h4>{% trans 'More documents like this one' %}</h4>
+<h4>{% trans 'Recommended documents' %}</h4>
```

Comment:
we needed a more general title for these similar documents since they now appear in multiple places. I went with "Recommended documents". If this term is going to be used in other contexts we can change it

Reply:
Another option is to move the heading out of this template so it can be customised where there's enough context to know how to phrase it

Reply:
since we are loading via htmx, we'd need to have separate templates so we can handle empty responses properly. Let me know if we need to do that. Using a common title is easier for now.